kofta/public/locales/nl/translation.json

```diff
@@ -0,0 +1,222 @@
+			"refresh": "microfoon lijst verversen",
+			"volume": "volume:"
+		}
+	},
+	"components": {
+		"avatar": {},
+		"backBar": {},
+		"blockedFromRoomUsers": {
+			"header": "Verbannen gebruikers",
+			"unban": "Verbanning ongedaan maken",
+			"noBans": "niemand is nog verbannen"
+		},
+		"bottomVoiceControl": {
+			"leaveCurrentRoomBtn": "Huidige kamer verlaten",
+			"confirmLeaveRoom": "Weet je het zeker dat je deze kamer wilt verlaten?",
+			"leave": "Verlaten",
+			"inviteUsersToRoomBtn": "Gebruikers uitnodigen",
+			"invite": "Uitnodigen",
+			"toggleMuteMicBtn": "Microfoon dempen schakelen",
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
+			"settings": "Instellingen",
+			"speaker": "Luidspreker",
+			"listener": "Luisteraar"
+		},
+		"deviceNotSupported": {
+			"notSupported": "Je toestel is momenteel niet ondersteund. Maak een",
+			"linkText": "kwestie aan op GitHub",
+			"addSupport": "wij zullen proberen uw toestel te laten ondersteunen."
+		},
+		"inviteButton": {
+			"invited": "uitgenodigd",
+			"inviteToRoom": "uitnodigen naar kamer"
+		},
+		"micPermissionBanner": {
+			"permissionDenied": "Toestemming geweigerd om toegang te krijgen tot je microfoon (mogelijk moet je naar de browserinstellingen gaan en de pagina opnieuw laden)",
+			"dismiss": "Sluiten",
+			"tryAgain": "probeer het opnieuw"
+		},
+		"keyboardShortcuts": {
+			"setKeybind": "zet toetsbinding",
+			"listening": "aan het luisteren",
+			"toggleMuteKeybind": "Microfoon dempen schakelen toetsbinding",
+			"togglePushToTalkKeybind": "druk-om-te-praten toetsbinding"
+		},
+		"userVolumeSlider": {
+			"noAudioMessage": "geen mogelijkheid om geluid te ontvangen door een onbekende reden"
+		},
+		"addToCalendar": {
+			"add": "Toevoegen aan kalender"
+		},
+		"wsKilled": {
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
+			"reconnect": "opnieuw verbinding maken"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "publiek",
+				"private": "privé",
+				"roomName": "kamer naam",
+				"roomDescription": "kamer beschrijving",
+				"descriptionError": "maximale grote 500",
+				"nameError": "moet tussen de 2 en 60 karakters lang zijn"
+			},
+			"invitedToJoinRoomModal": {
+				"newRoomCreated": "Nieuwe Kamer Gemaakt",
+				"roomInviteFrom": "Kamer uitnodiging van",
+				"justStarted": "Zij zijn net begonnen",
+				"likeToJoin": ", wil je deelnemen?",
+				"inviteReceived": "je bent uitgenodigd voor"
+			},
+			"editProfileModal": {
+				"usernameTaken": "gebruikers naam niet beschikbaar",
+				"avatarUrlError": "Ongeldige foto",
+				"avatarUrlLabel": "Github/Twitter avatar link",
+				"displayNameError": "lengte 2 tot 50 karakters",
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
```

Comment:
change "onderstrepingsteken" to "underscore"
again i've never heard of an "onderstrepingsteken"